cincinnati/src/plugins/mod.rs

```diff
@@ -387,7 +391,10 @@ where
     let mut runtime = tokio::runtime::Runtime::new()?;
 
     let timeout = match timeout {
-        None => return runtime.block_on(process(plugins, initial_io)),
+        None => {
+            let _span = get_tracer().start("plugins", None);
```

Comment:
Sorry, I had assumed you'd replace the other occurrences :smile: 

```suggestion
            let span = get_tracer().start("plugins", None);
```